git/pre-commit

```diff
@@ -2,6 +2,16 @@
 
 if [ "$NOLINT" ]; then exit 0; fi
 
+if [ ! $TRAVIS = true ]
```

Comment:
AFAIK to compare values you need user operant `==, !=` ...
@moylop260 I missing something in shell scripting ?